src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -201,14 +198,15 @@ export class CalcitePickListItem {
     ) : null;
 
     return (
-      <Host
-        selected={this.isSelected}
-        onKeydown={this.pickListKeyDownHandler}
-        role="checkbox"
-        aria-checked={this.isSelected}
-        tabindex="0"
-      >
-        <label class={CSS.label} onClick={this.pickListClickHandler}>
+      <Host selected={this.isSelected}>
+        <label
+          class={CSS.label}
+          onClick={this.pickListClickHandler}
+          onKeyDown={this.pickListKeyDownHandler}
+          role="checkbox"
+          aria-checked={this.isSelected}
+          tabindex="0"
```

Comment:
[Nitpick] We should be using `tabIndex`, no? We can do inventory on this in a follow-up issue.

Reply:
Yep. fixed! only found a couple